### PR TITLE
feat(Export): add push to registry. Do some cleanup.

### PR DESCRIPTION
**Pull request by lucbpz**
## What does this PR do?

- Updates the Export dialog to include the push to registry option.
- Cleanup actions - remove unused.

| Updated dialog | Cleanup actions |
| --- | --- |
| image | image |

### CODE REVIEW

File: ui/src/components/ExportDialog.tsx [hunk 1]
```diff
@@ -186,16 +193,43 @@ export default function ExportDialog({ open, onClose }: Props) {
...
+        {fromRadioValue === "push-registry" && (
+          <Stack pt={1} pb={2} pl={4} width="100%">
+            <Typography pb={2} variant="body2">
+              Push the volume content to a registry like DockerHub or GitHub
+              Container Registry.
+            </Typography>
+            <TextField
+              fullWidth
+              label="<registry>/<user>/<repo-name>:<tag>"
+              placeholder={defaultImageName}
```
Comment (felipecruz91) [inline ui/src/components/ExportDialog.tsx:213]: I'd like to use the placeholder to provide a clear and accurate example of what it'd like.

The `defaultImageName` is likely not to have the registry and user, that's why I think we should display something similar to:

```suggestion
              placeholder=`e.g. docker.io/john/${defaultImageName}`
```

**Review by felipecruz91 — CHANGES_REQUESTED** [review]
The same icon (arrow down) is being used for opposite actions

image

File: ui/src/components/ExportDialog.tsx [hunk 2]
```diff
@@ -186,16 +193,43 @@ export default function ExportDialog({ open, onClose }: Props) {
...
+        />
+        {fromRadioValue === "push-registry" && (
+          <Stack pt={1} pb={2} pl={4} width="100%">
+            <Typography pb={2} variant="body2">
+              Push the volume content to a registry like DockerHub or GitHub
+              Container Registry.
+            </Typography>
+            <TextField
+              fullWidth
+              label="<registry>/<user>/<repo-name>:<tag>"
```
Comment (felipecruz91) [inline ui/src/components/ExportDialog.tsx:212]: Add front-end validation to ensure that the input contains _at least_ the user, e.g:

✅ registry/johhn/repo-name:0.0.1 (registry is optional, if not specified, DockerHub will be used by default)

✅john/repo-name:0.0.1

❌ repo-name:0.0.1

**Review by felipecruz91 — COMMENTED** [review]
Can we keep the description of every radio button option always displayed even if the option is not checked? And make them appear in bold 😄 

image

File: ui/src/components/ExportDialog.tsx (hunk 1 above)
Comment (lucbpz) [reply ui/src/components/ExportDialog.tsx]: changed to avoid having a default value

File: ui/src/components/ExportDialog.tsx (hunk 2 above)
Comment (lucbpz) [reply ui/src/components/ExportDialog.tsx:225]: added some validation 👍

**Comment by lucbpz** [comment]
@felipecruz91 Changed also some styles in other dialogs, and to be consistent now that our designs are consistent I changed both Import and Export dialog to display the description of each radio button always. 

| export dialog | import dialog | clone dialog | delete dialog |
| -- | -- | -- | -- |
| image | image | image | image |

**Review by felipecruz91 — APPROVED** [review]
LGTM!

**Pull request merged**